web.config

```diff
@@ -0,0 +1,63 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+     This configuration file is required if iisnode is used to run node processes behind
+     IIS or IIS Express.  For more information, visit:
+
+     https://github.com/tjanczuk/iisnode/blob/master/src/samples/configuration/web.config
+-->
+
+<configuration>
+    <system.webServer>
+        <!-- Visit http://blogs.msdn.com/b/windowsazure/archive/2013/11/14/introduction-to-websockets-on-windows-azure-web-sites.aspx for more information on WebSocket support -->
+        <webSocket enabled="false" />
+        <handlers>
+            <!-- Indicates that the server.js file is a node.js site to be handled by the iisnode module -->
+            <add name="iisnode" path="server.js" verb="*" modules="iisnode"/>
+        </handlers>
+        <rewrite>
+            <rules>
+                <!-- Do not interfere with requests for node-inspector debugging -->
+                <rule name="NodeInspector" patternSyntax="ECMAScript" stopProcessing="true">
+                    <match url="^server.js\/debug[\/]?" />
+                </rule>
+
+                <!-- First we consider whether the incoming URL matches a physical file in the /public folder -->
+                <rule name="StaticContent">
+                    <action type="Rewrite" url="public{PATH_INFO}"/>
+                </rule>
+
+                <!-- All other URLs are mapped to the node.js site entry point -->
+                <rule name="DynamicContent">
+                    <conditions>
+                        <add input="{REQUEST_FILENAME}" matchType="IsFile" negate="True"/>
+                    </conditions>
+                    <action type="Rewrite" url="server.js"/>
+                </rule>
+            </rules>
+        </rewrite>
+
+        <!-- 'bin' directory has no special meaning in node.js and apps can be placed in it -->
+        <security>
+            <requestFiltering>
+                <hiddenSegments>
+                    <remove segment="bin"/>
+                </hiddenSegments>
+                <!-- file upload limit 100MB  -->
+                <requestLimits maxAllowedContentLength="104857600" />
```

Comment:
The content of this file was copied from automatically generated file on Azure and then this line `<requestLimits maxAllowedContentLength="104857600" />` was added to it to override default behaviour of blocking files with size greater than 30000000 bytes

Reply:
Do we really need to copy the default?

Reply:
For node.js apps that's the default expected, so had to copy it. I will do two commits as suggested 👍